README.md

```diff
@@ -1,9 +1,22 @@
 # ansible-gunicorn
 
-An Ansible role for installing and configuring gunicorn.
+An Ansible role for installing and configuring gunicorn. Provides the `Restart gunicorn` handler to reload gunicorn when you install/update your app.
 
 ## Role Variables
 
+- `gunicorn_start_on_vagrant` - sets whether to start on vagrant-mount (true) or when filesystems are mounted and a network interface comes up (false) (default: `false`)
```

Comment:
It may be more flexible to rename this to `gunicorn_start_on` and set the default value to something like `local-filesystems and net-device-up IFACE!=lo`.